src/browser/pages/vscode.ts

```diff
@@ -30,42 +39,45 @@ type NlsConfiguration = {
  *
  * Make sure to wrap this in a try/catch block when you call it.
```

Comment:
It's really a shame TypeScript has no support for throwing.

I gave this a thumbs up: https://github.com/microsoft/TypeScript/issues/13219

Thoughts on (I'm thinking we could do this later since we have a lot of `throw` around) using a sort of DocBlock syntax with `@throws`? Similar to the example in this issue: https://github.com/microsoft/TypeScript/issues/43528

We could also consider (at a later date, not in this PR) not using `throw` at all in our codebase and always returning something like Rust's `Result`.

Reply:
Ha funny, looks like I did too. I'll subscribe to it.

![image](https://user-images.githubusercontent.com/3806031/126375573-203639db-4fe0-44ae-a9de-e7822421e8cf.png)

I love either of those ideas! I'll add as a followup todo:

Reply:
https://github.com/cdr/code-server/issues/3825